Main/lib/sql/Oracle/createVdiControlTables.sql

```diff
@@ -12,6 +12,9 @@ CREATE TABLE VDI_CONTROL_&1..dataset (
 , type_version VARCHAR2(64)             NOT NULL
 , is_deleted   NUMBER       DEFAULT 0   NOT NULL
 , is_public    NUMBER       DEFAULT 0   NOT NULL
+, accessibility VARCHAR2(30)            NOT NULL
+, days_for_approval NUMBER            
```

Comment:
@aurreco-uga confirmed that making this `NOT NULL` and using `-1` to represent no limit is okay.